packages/txservice/src/shared/contracts.ts

```diff
@@ -3,14 +3,22 @@ import contractDeployments from "@connext/nxtp-contracts/deployments.json";
 import PriceOracleArtifact from "@connext/nxtp-contracts/artifacts/contracts/ConnextPriceOracle.sol/ConnextPriceOracle.json";
 import { ConnextPriceOracle as TConnextPriceOracle } from "@connext/nxtp-contracts/typechain";
 
+/**
+ * Helper to allow easy mocking
+ */
+export const getContractDeployments: any = () => {
+  return contractDeployments;
+};
+
 /**
  * A number[] list of all chain IDs on which a Connext Price Oracle Contracts
  * have been deployed.
  */
 export const CHAINS_WITH_PRICE_ORACLES: number[] = ((): number[] => {
   const chainIdsForGasFee: number[] = [];
-  Object.keys(contractDeployments).forEach((chainId) => {
-    const record = (contractDeployments as any)[chainId];
+  const _contractDeployments = getContractDeployments();
```

Comment:
what's with the underscore '_'? lol

Reply:
make sense, we have contractDeployements state in file